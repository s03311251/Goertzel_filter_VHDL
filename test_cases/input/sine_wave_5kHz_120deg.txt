3BB6
3B31
3AA6
3A15
397C
38DD
3838
378D
36DC
3625
3569
34A7
33E0
3313
3243
316D
3093
2FB5
2ED3
2DED
2D03
2C17
2B27
2A34
293F
2848
274E
2653
2556
2457
2358
2257
2157
2055
1F54
1E53
1D52
1C52
1B53
1A55
1958
185E
1765
166E
157A
1488
1399
12AE
11C5
10E1
1000
0F23
0E4A
0D76
0CA7
0BDC
0B17
0A57
099C
08E7
0838
078F
06EC
064F
05B9
052A
04A1
041F
03A4
0331
02C4
025F
0202
01AC
015E
0117
00D8
00A2
0073
004C
002D
0016
0007
0000
0002
000B
001D
0036
0058
0081
00B3
00EC
012E
0177
01C8
0220
0280
02E7
0356
03CC
