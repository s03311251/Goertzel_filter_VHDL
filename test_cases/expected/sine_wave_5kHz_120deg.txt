FFFDC520
FFFDBA60
